Allow become_flags on method dzdo Fixes #38685

one-t opened the pull request:
##### SUMMARY
Add Flags to the dzdo become method
Fixes #38685 

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
play_context

##### ANSIBLE VERSION
2.5.0


##### ADDITIONAL INFORMATION
The `dzdo` Become method does not currently accept flags set in `become_flags` - this pull request simply adds the flags to the `becomecmd`s for `dzdo`

badstreff committed:
added support for hidden users on macOS

webknjaz committed:
Upgrade PYPI badge to point to new Warehouse site

It should become default quite soon:
https://github.com/pypa/warehouse/milestone/1

bcoca committed:
fix displaying group vars in graph

bcoca committed:
minor fixes, also handle 'values' passed to dscl

Commit:
Allow become_flags on method dzdo Fixes #38685

Commit:
Allow become_flags on method dzdo Fixes #38685

bcoca committed:
minor fixes

sdodsley committed:
Add new Pure Storage module to discover facts for FlashArray (#37263)

sportebois committed:
Fix documentation for aws_ssm and aws_ssm_parameter_store (#38352)

Fix typos aws ssm vs aws_ssm and quotes

mattclay commented:
The sanity test failure is unrelated to this PR. The issue has been fixed. Tests have been restarted.

mattclay committed:
Work-around for get-pip.py install failure.

mattclay committed:
Disable fortios_* integration tests.

These tests require pyfg which depends on pip internals that are
no longer available under pip 10.

mattclay committed:
Mark rabbitmq_user test unstable.

mattclay committed:
Temporarily disable docker_secret test on RHEL.

Test does not pass on RHEL 7.4 with pip 10 installed.

mattclay committed:
Temporarily disable seboolean test on RHEL.

mattclay committed:
Temporarily disable sefcontext test on RHEL.

kenyon committed:
playbooks_async.rst: correct typo

mkrizek committed:
yum: docs: name is not required (#38758)

Akasurde committed:
influxdb: Add condition for retries argument (#38441)

'retries' argument was added to InfluxDBClient in version 4.1.0,
versions lower than this fails if retries are specified.
This fix adds a conditional to check if version is greater than equal to
4.1.0 and depending upon that adds 'retries' option.

Fixes: #38204

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

bcoca committed:
bool

Akasurde committed:
Minor fixes in terraform module (#38438)

This fix adds following -
* Check for None or blank value in project_path
* Make terraform binary path as required

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

ryansb committed:
Add docs to `AnsibleAWSModule.client` retry settings (#38689)

bocharsky-bw committed:
Remove duplicated "instead" word (#38800)

* Remove duplicated "instead" word

* Actually fix the message

bcoca committed:
centralize doc/config plugin lists (#38775)

* centralize doc/config plugin lists

also update list for generation in docsite
added note to ensure they are in sync

* updated shell page to list plugins

added some more docs hinting at plugins being configurable

* fix edit link for plugins

machacekondra committed:
ovirt_host_networks: Fix removing of network attachments (#38816)

mattclay committed:
Revert "Work-around for get-pip.py install failure."

This reverts commit fa466be33da819e0893bd14fe10d0f8e76e705a7.

bcoca committed:
BELOW THIS COMMENT

sivel committed:
Attempt 4: Prevent reparenting a block with itself (#38747)

* More concisely reparent, ensuring we don't go too shallow or too deep in this process. Fixes #38357

* More explicit reparenting, with a short circuit for a common case

* We need new_block to have a parent, otherwise we lose context with this approach

* Remove duplicate parent assignment

* Change callers of Block.copy to not use exclude_parent=True, when including the parent, exclude tasks

mattclay committed:
Use new vcenter simulator container location.

Commit:
Update dzdo become tests to use flags

Commit:
Update dzdo become tests to use flags

Commit:
change dzdo_flags to empty string

Commit:
change dzdo_flags to empty string

dagwieers committed:
Too much mail to keep up with vmware_guest, sorry

sivel committed:
Set memory-swap to memory (#38836)

bcoca committed:
fix exception on lookup ignore error

no log, so only_log ...

jctanner committed:
keep track of original inventory hostname in play context (#38818)

this change further enables the ansible-vcr project to collect fixtures per host in
situations where the host context is totally lost due to delegation.

* Keep some original pre-delegate values in connection options

sivel committed:
Skip self._parent on dynamic, defer to grandparent for attr lookup (#38827)

* Skip self._parent on dynamic, defer to grandparent for attr lookup

* Revert _inheritable

* Add tests for include inheritance from static blocks

Fixes #38037 #36194

gaudenz committed:
Fix zabbix_host for Zabbix Server versions < 3.0 (#38665)

The tls_* parameters are not present in Zabbix Server versions prior to
3.0. Thus the API response does not contain these keys and the
zabbix_host module failed. This commit adds checks if the parameters are
present in the API response and otherwise just completely ignores these
parameters. The documentation already states that they are not supported
for Zabbix Server versions below 3.0.

Commit:
Merge branch 'dzdo_flags' of https://github.com/one-t/ansible into dzdo_flags
